Lib/test/test_regrtest.py

```diff
@@ -2031,6 +2031,25 @@ def test_dev_mode(self):
         self.check_executed_tests(output, tests,
                                   stats=len(tests), parallel=True)
 
+    def test_import_from_tests(self):
```

Comment:
I suggest to rename the test to:

```suggestion
    def test_unload_tests(self):
```